form/management/commands/queue_worker.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+import form.queue
+
+
+class Command(BaseCommand):
+    help = 'Runs a form data SQS queue worker'
+
+    @staticmethod
+    def setup_logging():
+        logger = logging.getLogger()
```

Comment:
not if they all actually go to ``getLogger('')`` - ``__name__`` is no magic, you need to configure module-level logging to utilise it :P